test/quantization/test_quantize_fx.py

```diff
@@ -1695,15 +1695,23 @@ def forward(self, indices):
         model = M().eval()
         indices = torch.tensor([9, 6, 5, 7, 8, 8, 9, 2, 8, 6, 6, 9, 1, 6, 8, 8, 3, 2, 3, 6, 3, 6, 5, 7, 0, 8, 4, 6, 5, 8, 2, 3])
         quantized_node = ns.call_module(nnq.Embedding)
-        self.checkGraphModeFxOp(
-            model,
-            [[indices]],
-            QuantType.DYNAMIC,
-            quantized_node,
-            custom_qconfig=float_qparams_dynamic_qconfig
-        )
-
-    def test_qembedding_bag_module(self):
+        configs = [
+            (float_qparams_dynamic_qconfig, ns.call_module(nnq.Embedding)),
```

Comment:
What would happen in the case where the model has say a Linear layer and Embedding layer and we set the qconfig to `float_qparams_dynamic_qconfig `? We need to make sure in that case Linear doesn't get quantized as well.

Reply:
not sure if we are missing some checks, but this can be done in a separate pr

Reply:
Sounds good. Should we extend this test to include another static quant op in it (say linear) and test with different qconfig to make sure the right layers are quantized?

Reply:
you mean nn.linear with float_qparams_dynamic_qconfig right? will test this separately in a next pr.